CONTRIBUTING.md

```diff
@@ -28,8 +28,11 @@ Create a new [free developer instance](https://developer.atlassian.com/platform/
 
 Create a new [GitHub App](https://github.com/settings/apps), setting the following config:
 
-- **GitHub App name**: Anything you want, but it must be unique across GitHub.
-- **Homepage URL**: `https://DOMAIN`
+- **Domain**: The URL for the local domain obtained after tunneling.
```

Comment:
Can we make Domain not a list item? reads like its something i need to fill out :(

could just make it plain text or try quote text? I'll let you figure out what feels better  👍 

> **Domain**: The URL for the local domain obtained after tunneling.
>    - If the tunnel is created from `atlas`, then it would be something like `https://__MYDOMAIN__.public.atlastunnel.com`
>    - But if the tunnel is created from `ngrok`, then `https://XXXX-XXX-XXX-XXX-xX.ngrok.io`.

Reply:
Domain was separated out as a plain text.